run/test/specs/linked_device_profile_picture_syncs.spec.ts

```diff
@@ -1,55 +1,16 @@
-import { ANDROID_XPATHS } from '../../constants';
 import { androidIt, iosIt } from '../../types/sessionIt';
-import { sleepFor } from './utils';
-import { getAdbFullPath } from './utils/binaries';
+import { runOnlyOnAndroid, runOnlyOnIOS, sleepFor } from './utils';
 import { parseDataImage } from './utils/check_colour';
 import { linkedDevice } from './utils/link_device';
 import { SupportedPlatformsType, closeApp, openAppTwoDevices } from './utils/open_app';
```

Comment:
Would love a new line break here please 🙏

Reply:
Agreed! Added in